server/common.js

```diff
@@ -177,6 +179,49 @@ var getDate = function () {
 }
 exports.getDate = getDate;
 
+/**
+* return the current date
+*
+* @return {date}
+*/
+var getDateObject = function(){
+    return new Date();
+}
+exports.getDateObject = getDateObject;
+
+/**
+* return the time in day hours minutes and seconds format
+*
+* @return {string}
+*/
+var getTime = function(time){
```

Comment:
The work in this function seems unnecessary why can't it be implemented like the function above `getDate()`

Reply:
I am using this function to return back the amount of time the user has to wait in the locked message

Reply:
It is not the same as the function above